_docs/concepts/traffic-management/rules-configuration.md

```diff
@@ -148,6 +148,96 @@ route:
   weight: 75
 ```
 
+### Timeouts and Retries
+
+By default, the timeout for http requests is 15 seconds,
+but this can be overridden in a route rule as follows:
+
+```yaml
+destination: "ratings.default.svc.cluster.local"
+route:
+- tags:
+    version: v1
+httpReqTimeout:
+  simpleTimeout:
+    timeoutSeconds: 10
+```
+
```

Comment:
can you also mention this: This value can be overridden on a per-request basis if the application adds 
x-envoy-upstream-rq-timeout-ms header on outbound requests. The timeout must be specified in millisecond units.